functions.php

```diff
@@ -260,4 +260,13 @@ function getScoreboard($DB) {
     return $out;
 }
 
+function getMentors($users) {
```

Comment:
While you can technically use anything for the name of a parameter, it's generally a good idea for your parameters be named according to what they're supposed to represent. In this case, `getMentors()` should have the id of a usergroup for its parameter, so maybe a better name for the parameter would be `$group` or `$groupid`